### PR TITLE
Proxy methods for bindWithDelay/onBind

### PR DESCRIPTION
**Pull request by vanderlee**
Minimalist proxy methods for `bindWithDelay`/`onBind`.

You could still be backwards compatible if both proxy methods were aliasses of `onDelay`; any existing code should still work as is, but you could now add a selector like `onDelay`.

Also adjusted the index.html to use only onDelay; seems to work fine, but I'll look into qunit as you suggested.

### CODE REVIEW

**Comment by vanderlee** [comment]
Added a basic qunit unittest, just to get things started.

**Comment by bgrins** [comment]
Thanks for putting this together, and for adding the test! As I've thought about this a bit more, I think it might be best to leave `bindWithDelay` as a separate file, and support usage of onDelay through the documentation and demo.  

The reasoning here is that `bindWithDelay` will still work with jQuery pre-1.7 and if you are using a later version, you should probably be using the `on` api as encouraged by jQuery.  If we proxy the call it will not work with older versions.

I'm thinking we merge in all of the changes here, except for the proxy call in `onDelay.js`.  Does this make sense?

**Comment by vanderlee** [comment]
Good idea, hadn't thought of that. Still plenty of pre-1.7 users out there so it's probably best to keep them separate as you said.

File: onDelay.js [hunk 1]
```diff
@@ -52,4 +52,8 @@ $.fn.onDelay = function(events, selector, data, handler, timeout, throttle) {
     });
 };
 
+$.fn.onBind = $.fn.bindWithDelay = function(events, data, handler, timeout, throttle) {
```
Comment (bgrins) [inline onDelay.js]: Can you remove this change so that I can merge this PR in?

**Comment by vanderlee** [comment]
Done

**Comment by bgrins** [comment]
Thanks!  This fixes Issue #1

**Pull request merged**